docs/units/equivalencies.rst

```diff
@@ -347,6 +347,40 @@ and you want to know how big your pixels need to be to cover half an arcsecond::
     >>> (0.5*u.arcsec).to(u.micron, tel_platescale)  # doctest: +FLOAT_CMP
     <Quantity 18.9077335632719 micron>
 
+Reduced hubble constant/"little-h" Equivalencies
+------------------------------------------------
+
+The dimensionless version of the hubble constant - often known as "little h" -
+is a frequently-used quantity in extragalactic astrophysics.  It is also widely
+known as the bane of beginners' existence in such fields (See e.g., the title of
+`this paper <https://doi.org/10.1017/pasa.2013.31>`__, which also provides
+valuable advice on the use of little h).  Astropy provides an equivalency that
+helps keep this straight in at least some of these cases, by providing a way to
+convert to/from physical to "little h" units.  As an example::
+
+    >>> import astropy.units as u
+    >>> distance = 100 * (u.Mpc/u.h100)
+    >>> H0 = 70 * u.km/u.s / u.Mpc
```

Comment:
I vaguely remember `70 * (u.km/u.s / u.Mpc)` is more performant, is that right? I see that you apply that parenthesis in some places but it is not done consistently.

Nitpick: Spaces around operators.

Reply:
I would argue that inserting parenthesis like `(u.km / u.s) / u.Mpc` would be clearer than "space, no space" kind of notation. But I might be an outlier.

Reply:
To break the tie I asked a random astronomer who happened to be sitting next to me (@lea-hagen), and she said ``u.km/u.s / u.Mpc`` looked more natural, so leaving it as-is.

Reply:
There's always ``70 << u.km/u.s / u.Mpc``! 😄